infrastructure/kube/keep-prd/bcoin-statefulset.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: bcoin
+  namespace: default
+  labels:
+    app: bitcoin-node
+    type: bcoin
+spec:
+  replicas: 2
+  serviceName: bcoin
+  volumeClaimTemplates:
+  - metadata:
+      name: bcoin-data
+    spec:
+      storageClassName: bcoin
+      accessModes: [ReadWriteOnce]
+      resources:
+        requests:
+          storage: 450Gi
```

Comment:
Have we captured somewhere that we should be monitoring this to make sure we can react before we run out of disk?

Reply:
We'll have an alert policy for disk space in GCP Monitoring, otherwise no.

Reply:
I think that's all we need, just making sure it's on the radar.